src/abstractdatatable/abstractdatatable.jl

```diff
@@ -379,6 +378,45 @@ function StatsBase.describe(io, dt::AbstractDataTable)
     end
 end
 
+function StatsBase.describe{T <: Real}(io::IO, X::Vector{Union{T, Null}})
+    nullcount = count(isnull, X)
+    pnull = 100 * nullcount/length(X)
+    if pnull != 100 # describe will fail if dropnull returns an empty vector
+        describe(io, collect(Nulls.skip(X)))
+    else
+        println(io, "Summary Stats:")
+        println(io, "Length:         $(length(X))")
+        println(io, "Type:           $T")
+    end
+    println(io, "Number Missing: $(nullcount)")
+    @printf(io, "%% Missing:      %.6f\n", pnull)
+    return
+end
+
+function StatsBase.describe{T}(io::IO, X::Vector{Union{T, Null}})
+    nullcount = count(isnull, X)
+    pnull = 100 * nullcount/length(X)
+    println(io, "Summary Stats:")
+    println(io, "Length:         $(length(X))")
+    println(io, "Type:           ?$T")
+    println(io, "Number Unique:  $(length(unique(X)))")
+    println(io, "Number Missing: $(nullcount)")
+    @printf(io, "%% Missing:      %.6f\n", pnull)
+    return
+end
+
+function StatsBase.describe{T}(io::IO, X::CategoricalVector{T})
```

Comment:
Ah. Mainly because I hadn't noticed I had made the generic method too restrictive with `function StatsBase.describe{T}(io::IO, X::Vector{Union{T, Null}})` when it could/should be `X::AbstractVector{Union{T, Null}})`. I'll push an update